### ISSUE TITLE
Config Generator

### ISSUE THREAD

**Issue by pmarkowsky** [northpolesec/santa#10]
Crafting a configuration profile is tedious, difficult to get right, and the documentation for all the keys is difficult to write.

We could host a config generator site on santa.dev as a form with instructions for each key that then produces a valid plist config. This will require a fair bit of JavaScript but GH pages supports hosting JS files so this should be OK.

As well as a form being much simpler to use, we can document each key in-line with the form.

**Comment by arubdesu** [comment]
Agreed on this being a barrier to entry. On this note, there is a community-led (life-support-maintained) 'full' app project called [Profile Creator](https://github.com/ProfileCreator/ProfileCreator) and another 'fat' app [iMazing profile-editor](https://imazing.com/profile-editor) that make the experience more palatable, but 'prior art' for it being in a web page is (similarly zoomed in on a specific use case) 'dock master' https://techion.com.au/blog/2015/4/28/dock-master (it was 2015, maybe we can cut them some slack on naming 😅).
